internal/service/advancedclustertpf/testdata/TestAccMockableAdvancedCluster_replicasetAdvConfigUpdate/02_02_PATCH__api_atlas_v2_groups_{groupId}_clusters_{clusterName}_processArgs_2024-08-05.json

```diff
@@ -1,7 +1,6 @@
 {
  "changeStreamOptionsPreAndPostImagesExpireAfterSeconds": 100,
  "defaultWriteConcern": "majority",
- "minimumEnabledTlsProtocol": "TLS1_0",
```

Comment:
This property is now not present in the PATCH because there is no change from TLS1_1 to TLS1_0. Now it preserves TLS1_2 in both steps.